core/db/migrate/20150826002555_convert_habtm_to_hmt_for_taxon_prototypes.rb

```diff
@@ -0,0 +1,15 @@
+class ConvertHabtmToHmtForTaxonPrototypes < ActiveRecord::Migration
+  def up
+    add_column :spree_taxons_prototypes, :created_at, :datetime
```

Comment:
Actually, I cherry-picked the wrong commit. If you check the branch name, this was actually correct. Through an interactive rebase, I've removed the commits that aren't specific to this PR. Apologies.